packages/widget-reorder/src/control.tsx

```diff
@@ -1,117 +1,123 @@
-                borderRadius: '3px',
-              }}
-              {...provided.droppableProps}
-              ref={provided.innerRef}
-            >
-              {data.map((item, i) => (
-                <Draggable
-                  key={item[fieldId]}
-                  draggableId={item[fieldId]}
-                  index={i}
-                >
-                  {(provided, snapshot) => (
-                    <div
-                      ref={provided.innerRef}
-                      {...provided.draggableProps}
-                      {...provided.dragHandleProps}
-                      style={{
-                        padding: '1rem',
-                        opacity: snapshot.isDragging ? 0.6 : 1,
-                        boxShadow: snapshot.isDragging ?  '0 4px 16px 0 rgba(0,0,0,0.2)' : '0 2px 6px 0 rgba(0,0,0,0.2)',
-                        background: '#fff',
-                        borderRadius: '3px',
-                        marginBottom: '0.5rem',
-                        ...provided.draggableProps.style
-                      }}
-                    >
-                      {fieldDisplay.map(fieldName => item[fieldName]).join(' ')}
+    public render() {
+      const { field } = this.props
+      const { data } = this.state
+  
+      const fieldId: string = field.get('id_field')
+      const fieldDisplay: string[] = field.get('display_fields')
+
+      if (data.length === 0) return <div>loading...</div>
+      return (
+        <DragDropContext onDragEnd={this.handleDragEnd}>
+          <Droppable droppableId="droppable">
+            {(provided, snapshot) => (
+              <div
+                style={{
+                  padding: '1rem',
+                  background: snapshot.isDraggingOver ? 'lightblue' : '#dfdfe3',
+                  borderRadius: '3px',
+                }}
+                {...provided.droppableProps}
+                ref={provided.innerRef}
+              >
+                {data.map((item, i) => (
+                  <Draggable
+                    key={item[fieldId]}
+                    draggableId={item[fieldId]}
+                    index={i}
+                  >
+                    {(provided, snapshot) => (
+                      <div
+                        ref={provided.innerRef}
+                        {...provided.draggableProps}
+                        {...provided.dragHandleProps}
+                        style={{
+                          padding: '1rem',
+                          opacity: snapshot.isDragging ? 0.6 : 1,
+                          boxShadow: snapshot.isDragging ?  '0 4px 16px 0 rgba(0,0,0,0.2)' : '0 2px 6px 0 rgba(0,0,0,0.2)',
+                          background: '#fff',
+                          borderRadius: '3px',
+                          marginBottom: '0.5rem',
+                          ...provided.draggableProps.style,
+                        }}
+                      >
+                      <ListElement item={extract(item, ...fieldDisplay)}/> 
```

Comment:
On line 26, we've already extract fieldDisplay from data, so we won't have to do this again here

```
      const data = result.payload.response.hits.map(payload => {
        return extract(payload.data, ...fieldsToBeExtracted)
      })
```